docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: AWS Console and CLI access
+description: How to access AWS with Teleport.
```

Comment:
I'm not sure this title and description provides enough information.

We _already_ have the ability to access AWS Console and CLI with Teleport prior to the roles anywhere integration, so it's not clear what the integration provides or why I as a docs reader would even end up on this page and not one of the other guides.

Reply:
The "tip" box should help the user understand which methods exist and which one they should pick.
I moved it up so that it's the first information the user sees (right after the page's title).